capstone-frontend/src/components/ListPage.jsx

```diff
@@ -79,49 +103,78 @@ class ListPage extends Component {
         <h1>Loading...</h1>
       </div>)
       :
-      (this.state.empty
+      ((this.state.gBooks.length === 0)
```

Comment:
Nice job checking `gBooks.length` here, enabling you to remove `empty` from the state!